Cargo.toml

```diff
@@ -66,24 +67,29 @@ clap = { version = "4.5", optional = true, features = ["derive"] }
 num = { version = "0.4", default-features = false, features = ["alloc", "libm"] }
 num-complex = { version = "0.4", default-features = false }
 rand = { version = "0.8", default-features = false }
+rand_chacha = { version = "0.3", default-features = false }
 rand_core = { version = "0.6", default-features = false }
-rand-utils = { version = "0.11", package = "winter-rand-utils", optional = true }
 rayon = { version = "1.10", optional = true }
 serde = { version = "1.0", default-features = false, optional = true, features = ["derive"] }
 sha3 = { version = "0.10", default-features = false }
 thiserror = { version = "2.0", default-features = false }
-winter-crypto = { version = "0.11", default-features = false }
-winter-math = { version = "0.11", default-features = false }
-winter-utils = { version = "0.11", default-features = false }
+winter-air = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-crypto = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-prover = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-verifier = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-math = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-fri = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
+winter-maybe-async = {git = 'https://github.com/Al-Kindi-0/winterfell', branch = 'al-zk' }
```

Comment:
Are we using `winter-fir` and `winter-maybe-async` somewhere? (upon a quick scan I couldn't see where they are used).

Reply:
Indeed, removed now